test/artifacts/MySupportInfo.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Totally not a blocker, but this file naming is surprising to me in two ways, just wondering your thoughts:

1. I typically see this kind of thing called a `fixture`, is the convention of `artifacts` from another school of thought?
2. All the other files are kebab, this is camel.  While this might not bite anyone on a regular basis, avoiding upper case in file names means you don't worry about case-insensitive file systems.  Here is both goes counter to the other files and has uppercase in the filename.  Is there a reason?

Reply:
will fix